src/ert/simulator/batch_simulator_context.py

```diff
@@ -218,20 +247,37 @@ def results(self) -> List[Optional[Dict[str, "npt.NDArray[np.float64]"]]]:
 
         return res
 
-    def job_status(self, iens: int) -> Optional["JobStatus"]:
+    def job_status(self, iens: int) -> Optional["DeprecatedJobStatus"]:
```

Comment:
I think the commit message should have some explanation on this, since we are not really removing the translation, only shuffling it around internally to be able to stay compatible with Everest.

The commit subject is okay, but the commit body deserves a paragraph.

Reply:
Fixed 🔧

Reply:
(remember that any text put in the pull request is lost after merge, so the git commit body is always a better place)